sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -279,21 +282,21 @@ def get_estimated_size_and_observables(self, value, nested=False):
 
   def encode_to_stream(self, value, stream, nested):
     t = type(value)
-    if t is NoneType:
+    if value is None:
       stream.write_byte(NONE_TYPE)
     elif t is int:
       stream.write_byte(INT_TYPE)
       stream.write_var_int64(value)
     elif t is float:
       stream.write_byte(FLOAT_TYPE)
       stream.write_bigendian_double(value)
-    elif t is str:
-      stream.write_byte(STR_TYPE)
+    elif t is bytes:
+      stream.write_byte(BYTES_TYPE)
       stream.write(value, nested)
-    elif t is six.text_type:
-      unicode_value = value  # for typing
-      stream.write_byte(UNICODE_TYPE)
-      stream.write(unicode_value.encode('utf-8'), nested)
+    elif t is unicode:
+      text_value = value  # for typing
```

Comment:
Can you use the same `try: unicode except: unicode = str` in the corresponding part of the .pxd file so that the type annotation directive for `text_value` is respected?

Reply:
I tried this, but it throws a 'not a type' error for `unicode`.